addons/l10n_account_edi_ubl_cii_tests/tests/common.py

```diff
@@ -2,6 +2,7 @@
 import base64
 
 from freezegun import freeze_time
+from collections import Counter
```

Comment:
unnecessary diff

Reply:
What? No.

This is because the default order on invoice lines changed (removed the sequence), so the assertion values are not in the same order anymore.
But we don't care about the order so I just used a counter.